### PR TITLE
Webextension

### PR DESCRIPTION
**Pull request by pastak**
It will run on Mozilla Firefox.

https://wiki.mozilla.org/WebExtensions

### CODE REVIEW

**Comment by pastak** [comment]
[Gyazo](https://gyazo.com/6207d897e117065649a3607111bde4d4)

File: src/manifest.json [hunk 1]
```diff
@@ -24,11 +24,11 @@
       }
   },
   "web_accessible_resources": [
-    "fonts/icomoon.eot",
-    "fonts/icomoon.svg",
-    "fonts/icomoon.ttf",
-    "fonts/icomoon.woff",
-    "fonts/icomoon.woff2"
+    "/fonts/icomoon.eot",
```
Comment (rakusai) [inline src/manifest.json]: why do we need this only for fonts? why not for `/icon`

File: src/statics/menu.css [hunk 2]
```diff
@@ -121,12 +121,16 @@ body.gyazo-select-element-mode *:not(.gyazo-crop-select-element):not(.gyazo-menu
...
...BwcFLMUHR0UAs2zDRERDbMMEREM/LwLFQYKChCbECQJCQkQmwYLBgJtCA4EBgYLmwoYBwYHCpsECAP+NwUKBQ8JCVkJIA8PCQlZBhMLAWYEBwQKBwZaBhgLCgcGWgQOCAKRIxmzGCMjGLMZI/1KFg+zEBYWELMPFgKWDg6bFC0LDAwUmxQtDAQD/aoICZsMGwYHBwybDBsHAgIB9wQDWgoqExMLC1kLKhMMDv6wAgJaBhgLCgcGWgYYCwcIzhwUFBwcFBQcEhIMDBISDAwS7wwLECQJWQoKEBAkCVoDAwF1CAcLGAZaBgcKCxgGWgIC/esDAwghD5sPCAgJIA+bCgsCeAICBhgLmwoHBgcYCpsHCAABAAAAAAAAGpgA+18PPPUACwQAAAAAANIO+WIAAAAA0g75YgAA/9sD3QO7AAAACAACAAAAAAAAAAEAAAPA/8AAAAQAAAAAAAPdAAEAAAAAAAAAAAAAAAAAAAAKBAAAAAAAAAAAAAAAAgAAAAQAAKsEAABkBAAAZAQAAGQEAACGBAAAEAAAAAAACgAUAB4AZACMALAAyAEqAl4AAAABAAAACgDSAAwAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAAADgCuAAEAAAAAAAEABwAAAAEAAAAAAAIABwBgAAEAAAAAAAMABwA2AAEAAAAAAAQABwB1AAEAAAAAAAUACwAVAAEAAAAAAAYABwBLAAEAAAAAAAoAGgCKAAMAAQQJAAEADgAHAAMAAQQJAAIADgBnAAMAAQQJAAMADgA9AAMAAQQJAAQADgB8AAMAAQQJAAUAFgAgAAMAAQQJAAYADgBSAAMAAQQJAAoANACkaWNvbW9vbgBpAGMAbwBtAG8AbwBuVmVyc2lvbiAxLjAAVgBlAHIAcwBpAG8AbgAgADEALgAwaWNvbW9vbgBpAGMAbwBtAG8AbwBuaWNvbW9vbgBpAGMAbwBtAG8AbwBuUmVndWxhcgBSAGUAZwB1AGwAYQByaWNvbW9vbgBpAGMAbwBtAG8AbwBuRm9udCBnZW5lcmF0ZWQgYnkgSWNvTW9vbi4ARgBvAG4AdAAgAGcAZQBuAGUAcgBhAHQAZQBkACAAYgB5ACAASQBjAG8ATQBvAG8AbgAuAAAAAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==) format('woff');
```
Comment (rakusai) [inline src/statics/menu.css]: ah.

File: src/manifest.json (hunk 1 above)
Comment (pastak) [inline src/manifest.json]: icons is not embeded

File: src/statics/menu.css (hunk 2 above)
Comment (pastak) [inline src/statics/menu.css]: Firefox Extension cannot get contents on `chrome-extension:XXX` which written on `web_accessible_resources`. Maybe solve by stable release?

File: src/manifest.json (hunk 1 above)
Comment (pastak) [inline src/manifest.json]: This change is for Firefox doesn't yet support to resolve path ( need `/` at first ).

**Comment by pastak** [comment]
It runs on Firefox Nightly or Firefox Developer Edition now. Stable release is scheduled on Firefox 38 will release 2016/8/3.

We can prepare for release on AMO.

**Comment by pastak** [comment]
It can run on least Firefox 47. Firefox 45 is able to install WebExtension but https://bugzilla.mozilla.org/show_bug.cgi?id=1210583 is solved on Firefox47.

**Comment by pastak** [comment]
If you use Firefox Beta v47.0b1, you can try it out: https://www.dropbox.com/s/y2f7zsqydz3f38o/gyazo-2.3.0.xpi?dl=0

**Pull request merged**